airbyte-ci/connectors/pipelines/pipelines/helpers/utils.py

```diff
@@ -326,3 +327,28 @@ def fail_if_missing_docker_hub_creds(ctx: click.Context) -> None:
         raise click.UsageError(
             "You need to be logged to DockerHub registry to run this command. Please set DOCKER_HUB_USERNAME and DOCKER_HUB_PASSWORD environment variables."
         )
+
+
+def java_log_scrub_pattern(secrets_to_mask: List[str]) -> str:
```

Comment:
Could you please add a unit test for this? 🙏

Reply:
Yes! sorry it slipped my mind